proposal.tex

```diff
@@ -37,6 +48,12 @@ \subsection{BPS Batch }\label{sec:bpsbatch}
 This would also cover DMS-REQ-0119, DMS-REQ-0125, DMS-REQ-0128, DMS-REQ-0123, DMS-REQ-0127.
 
 
+\subsubsection{Non Rubin code} \label{sec:othercode}
+Users may wish to run arbitrary code on Rubin images.
+It should be trivial to provide a butler recipe to get an image or set of images to the users space and allow them to run any code which takes FITS images. Done in a container this could be done for a large volume of images - it would have to get past  the resource allocation committee of course.
+Run in this mode no provenance would be kept.  (See \secref{sec:gen3})
```

Comment:
We would have to be able to ingest the generated output files and there would be some provenance in the sense that we would store the graph provenance and would know the dataId. We wouldn't have the full provenance of tracking software versions and configuration.

Reply:
I imagine if butler is storing the outputs we'd need to just assume that we provide a dummy formatter that won't read the data. This assumes that people are specifying an output dataset type at least. If they don't want to even tell us about the output files we have to copy them somewhere (VOSpace?) and let them sort out what to do with thousands of files.

Reply:
added some text ..